package.json

```diff
@@ -35,13 +36,15 @@
     "eslint": "^2.1.0",
     "eslint-plugin-json": "^1.2.0",
     "eslint-plugin-mocha": "^2.0.0",
-    "eslint-plugin-react": "^3.16.1",
+    "eslint-plugin-react": "^4.1.0",
```

Comment:
I've actually encountered race conditions with this setup before, I get warnings randomly when installing this package that the peer dependencies have not been met.  Having purely just peer dependencies works in both npm 2 and npm 3, this isn't an actual issue.  If using npm 3 and using this as a dependency then yes, it is expected that you must manually install the peer dependency packages; that is its intended purpose and functionality.